admin/services/generate-pins.service.js

```diff
@@ -0,0 +1,19 @@
+const pupilDataService = require('../services/data-access/pupil.data.service')
+
+const generatePinService = {
```

Comment:
Where is the actual method to generate pins then?

Reply:
@jon-shipley  that's not part of the current ticket.
This is just scaffold UI
https://agilefactory.visualstudio.com/MTC/_workitems?id=16341&_a=edit